klaviyo/exceptions.py

```diff
@@ -0,0 +1,48 @@
+import simplejson
+
+
+class KlaviyoException(Exception):
+    def __init__(self, message):
+        super(KlaviyoException, self).__init__(message)
+        self.message = message
+
+
+class KlaviyoConfigurationException(KlaviyoException):
+    pass
+
+
+class KlaviyoApiException(Exception):
```

Comment:
Probably a nit but should this be `KlaviyoAPIException`?

API vs Api

the other classes seem to use API (KlaviyoAPI, KlaviyoAPIFixture), not a big deal but consistency is good.

Reply:
totally 👍